tests/dummy/app/templates/docs.hbs

```diff
@@ -24,6 +24,9 @@
 
     {{nav.section "Components"}}
     {{nav.item "Links" "docs.links"}}
+
+    {{nav.section "Migrating"}}
```

Comment:
Do we want to call this migrating? Do we want to keep it more general? What other things would go in this section?

Reply:
Yes, because in this section we going to have things like **deprecations**, a **guide to upgrade to v1.0**, and stuff like that. Does it makes sense to you?

Reply:
I think upgrading is good...I am trying to figure a better word...For the time being I think Migrating is fine, we can always change the section name eventually.